.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -11,3 +11,6 @@ lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
 source "${lib_script}"
 
 gen_clean_arch || info "Arch cleanup scripts failed"
+
+info "clean up /tmp"
+sudo sh -c 'rm -rf /tmp/*'
```

Comment:
Hmm, that feels a bit too 'hard' to me - the system may have other tmp files down there it is using (given this is a baremetal server). I fear you will corrupt the server long term.

Also, have we identified which bit of the ci/test scripts is leaving the rubbish in `/tmp`? Some time ago I spent some time making sure the test framework did not do that - if we can identify the 'dirty tests' and clean them up, that would be better :-)

Reply:
thanks @Pennyzct , I agree with @grahamwhaley - remove all file in `/tmp` is prone to break something